Extension/lib/content-script/wrappers.js

```diff
@@ -118,284 +203,290 @@ var injectPageScriptAPI = function (injectApi) { // jshint ignore:line
-        };
-    }
+            RealRTCPeerConnection.prototype.setConfiguration = function (configuration) {
+                configuration = protectConfiguration(configuration);
+                // Call the real method first, so that validates the configuration
+                realSetConfiguration(this, configuration);
+                checkConfiguration(this, configuration);
+            };
+        }
 
-    function WrappedRTCPeerConnection(configuration, arg) {
+        function WrappedRTCPeerConnection(configuration, arg) {
 
-        if (!(this instanceof WrappedRTCPeerConnection)) {
-            return RealRTCPeerConnection();
-        }
+            if (!(this instanceof WrappedRTCPeerConnection)) {
+                return RealRTCPeerConnection();
+            }
+
+            configuration = protectConfiguration(configuration);
 
-        configuration = protectConfiguration(configuration);
+            /**
+             * The old webkitRTCPeerConnection constructor takes an optional second argument and we must pass it.
+             */
+            var connection = new RealRTCPeerConnection(configuration, arg);
+            checkConfiguration(connection, configuration);
+            return connection;
+        }
 
-        /**
-         * The old webkitRTCPeerConnection constructor takes an optional second argument and we must pass it.
-         */
-        var connection = new RealRTCPeerConnection(configuration, arg);
-        checkConfiguration(connection, configuration);
-        return connection;
-    }
+        WrappedRTCPeerConnection.prototype = RealRTCPeerConnection.prototype;
 
-    WrappedRTCPeerConnection.prototype = RealRTCPeerConnection.prototype;
+        var boundWrappedRTCPeerConnection = WrappedRTCPeerConnection.bind();
+        copyProperties(RealRTCPeerConnection, boundWrappedRTCPeerConnection, ["caller", "generateCertificate", "name", "prototype"]);
+        RealRTCPeerConnection.prototype.constructor = boundWrappedRTCPeerConnection;
 
-    var boundWrappedRTCPeerConnection = WrappedRTCPeerConnection.bind();
-    injectApi.copyProperties(RealRTCPeerConnection, boundWrappedRTCPeerConnection, ["caller", "generateCertificate", "name", "prototype"]);
-    RealRTCPeerConnection.prototype.constructor = boundWrappedRTCPeerConnection;
+        if ("RTCPeerConnection" in window) {
+            window.RTCPeerConnection = boundWrappedRTCPeerConnection;
+        }
+        if ("webkitRTCPeerConnection" in window) {
+            window.webkitRTCPeerConnection = boundWrappedRTCPeerConnection;
+        }
+    };
 
-    if ("RTCPeerConnection" in window) {
-        window.RTCPeerConnection = boundWrappedRTCPeerConnection;
+    if (shouldOverrideWebSocket) {
+        overrideWebSocket();
     }
-    if ("webkitRTCPeerConnection" in window) {
-        window.webkitRTCPeerConnection = boundWrappedRTCPeerConnection;
+
+    if (shouldOverrideWebRTC) {
+        overrideWebRTC();
     }
-};
+}
```

Comment:
Hm, I don't know how to do that. `injectPageScriptAPI` contains code which injects itself into a page, so this function should contain all code.